pkg/streamingpromql/testdata/ours/aggregators.test

```diff
@@ -31,3 +31,41 @@ load 1m
 eval range from 1m to 1m30s step 1s sum(some_metric_with_staleness)
   # Should return no results.
 
+clear
+
+# Test native histogram aggregations
+load 1m
+	single_histogram{label="value"} {{schema:0 sum:2 count:4 buckets:[1 2 1]}} {{sum:2 count:4 buckets:[1 2 1]}}
+	single_histogram{label="value2"} {{schema:1 sum:5 count:5 buckets:[1 3 1]}} {{schema:0 sum:2 count:4 buckets:[1 2 1]}}
+	single_histogram{label="value2", another="value"} {{schema:1 sum:10 count:9 buckets:[3 3 3]}} {{schema:2 sum:2 count:4 buckets:[1 2 1]}}
+
+eval instant at 0m sum(single_histogram)
+	{} {{sum:17 count:18 buckets:[5 12 1]}}
```

Comment:
[nit] Might be good to make the schema here explicit - took me a second to realise this is also testing the case that different series have samples with different schemas.

(same applies below)